aep/general/0231/aep.md.j2

```diff
@@ -126,24 +115,66 @@ message BatchGetBooksResponse {
 }
 ```
 
-- The response message **must** include one repeated field corresponding to the
-  resources being retrieved.
+- The response message **must** include one repeated field where its items may
+  either be a retrieved resource or an error structure describing an error that
+  occurred attempting to retrieve the resource. The error alternative would
+  only be present for non-transactional batch gets.
 
 {% tab oas %}
 
-- The response schema **must** be an array with each item containing one of the
-  requested resources.
+- The response schema **must** be `type: object` with a single array property
+  with each item containing one of the requested resources.
```

Comment:
This seems substantially different from the proto guidance:

1.  OAS responses _only_ have an array, where proto responses just have to _include_ the repeated field.
2.  The OAS array just has the requested resource; the proto repeated field has either the resource or an error object. 

These both seem like they should be the same across IDLs - no?

Reply:
They should be the same. I think the OAS description is what we want. I'll gladly fix the proto description if someone can tell me what it should be.